system_setup/models/wslconfadvanced.py

```diff
@@ -18,14 +18,15 @@
 import attr
 
 from subiquitycore.utils import run_command
+from .wslconfbase import WSLConfigurationBase
 
 log = logging.getLogger('subiquity.models.wsl_configuration_advanced')
 
 # TODO WSL: Remove all common attributes with wslconfbase
 
 
 @attr.s
-class WSLConfigurationAdvanced(object):
+class WSLConfigurationAdvanced(WSLConfigurationBase):
```

Comment:
No inheritance from Base

Reply:
See first comment about inheritance and remove it from your first commit.

Reply:
fixed